apps/api/src/domain/models/shared/functional/common/not.spec.ts

```diff
@@ -0,0 +1,55 @@
+import not from './not';
```

Comment:
Interesting

Reply:
That reminds me- I need to make this work as an anti-typeGuard when you pass it a typeGuard.